plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -219,7 +219,11 @@ protected function deleteCuePoints(Criteria $c)
 		CuePointPeer::setUseCriteriaFilter(true);
 
 		foreach($cuePoints as $cuePoint)
+		{
+			$cuePoint->reload();
```

Comment:
@tan-tan-kanarek do you think it's better to call $cuePoint->setStatus(CuePointStatus::DELETED) here instead?